### ISSUE TITLE
Export / import should let the user choose the file location

### ISSUE THREAD

**Issue by anthonydahanne** [michaelkourlas/voipms-sms-client#147]
[tested on 0.5.3 from f-droid]
Switching to a new Android phone (and configuring everything form scratch), I thought I could use the export / import feature, but could not.

On the old device, export said it worked correctly, but from the Android file transfer application on mac, I could not see Android/data/net.kourlas.voipms_sms/backup_export.db
So I ended up using adb, and I could retrieve the file with : 
```
./adb pull /storage/self/primary/Android/data/net.kourlas.voipms_sms/files/backup_export.db ~/Downloads/
```

On the new device, import said the file could not be located, even if I successfully copied the file I retrieved previously to the said location
<img width="621" alt="screen shot 2017-11-01 at 16 49 26" src="https://user-images.githubusercontent.com/60938/32297171-ad51221c-bf24-11e7-9890-a55137a2e846.png">
![screenshot_20171101-164723](https://user-images.githubusercontent.com/60938/32297176-b3e7c888-bf24-11e7-906a-c04d4c5df9f7.png)

All in all, I could not import my DB...
I believe it would work better with a file / directory picker, or, just a text input field that would allow the user to choose the folder - because I believe wrong permissions or wrong assumptions on the folder /storage/emulated/0/ are causing the feature difficult to use, if usable at all in my case...

**Comment by anthonydahanne** [comment]
haha ! looking at the source code, I found out that... the file is differently named between import and export !
I renamed it and it worked !
Super confusing though + I had issues to extract it, and I had to use adb, so I keep the issue open !

**Comment by michaelkourlas** [comment]
This has been implemented and will be in 0.6.0.
